index.html

```diff
@@ -44,7 +44,163 @@ <h1>I'm Javi,<br>
+          <div>
+            <img class="border-10px fill-100" src="assets/images/Snapshoot Portfolio 4.svg" alt="Canopy project snapshot">
+          </div>
+          <h2 class="p10px">Multi-post<br>
+            stories</h2>
+          <div class="flex aic">
+            <h4 class="p10px">Canopy</h4>
+            <img src="assets/images/Counter.svg" alt="Spacer">
+            <h5 class="p10px">Back end dev</h5>
+            <img src="assets/images/Counter.svg" alt="Spacer">
+            <h5 class="p10px">2022</h5>
+          </div>
+          <p class="p10px m0">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="xul">
+            <li class="mh10px p10px border-10px bg-tag"><h6>html</h6></li>
+            <li class="mh10px p10px border-10px bg-tag"><h6>css</h6></li>
+            <li class="mh10px p10px border-10px bg-tag"><h6>javaScript</h6></li>
+          </ul>
+          <div>
+            <button>See project</button>
+          </div>
+        </div>
+      </section>
+    </article>
+    <article class="bg-box border-tr pv5">
+      <section class="p5vh">
+        <h2>About<br>
+          Myself</h2>
+        <p class="mv12px">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a class="mv12px" href="https://ly.enddesign.co/intro"><h3>Let's connect</h3></a>
+        <ul class="xul">
+          <li class="pr20px">
+            <a href="http://">
+              <img class="icon" src="assets/images/Linkedin icon.svg" alt="Linkedin profile @javierjhm">
+            </a>
+          </li>
+          <li>
+            <a href="http://">
+              <img class="icon" src="assets/images/Github icon.svg" alt="Github profile @wickathou">
+            </a>
+          </li>
+        </ul>
+        <button class="m0">Get my resume</button>
+      </section>
+      <section class="p5vh">
+        <ul class="xul flex-column">
+          <li class=" border-bottom-line">
+            <div class="flex aic jcb">
+              <h2 class="fz20px">Language</h2>
+              <img src="assets/images/chevron.svg" alt="">
+            </div>
+            <div class="grid-container-list pv20px">
+              <div class="flex aic p10px bg-list border-10px">
+                <img class="pr10px icon-lg" src="assets/images/javascript circle.svg" alt="">
+                <p>javaScript</p>
+              </div>
+              <div class="flex aic p10px bg-list border-10px">
+                <img class="pr10px icon-lg" src="assets/images/html circle.svg" alt="">
+                <p>HTML</p>
+              </div>
+              <div class="flex aic p10px bg-list border-10px">
+                <img class="pr10px icon-lg" src="assets/images/css circle.svg" alt="">
+                <p>CSS</p>
+              </div>
```

Comment:
- [ ] I would suggest you make this technology part also have a `ul` tag as the container tag and each technology i.e ` JavaScript`, `HTML`, ans `CSS` have a `li` tag wrapped inside the `ul` tag